client/my-sites/email/email-management/home/email-mailbox-action-menu.jsx

```diff
@@ -89,32 +85,15 @@ const getTitanMenuItems = ( {
 
 	return [
 		{
-			href: getTitanEmailUrl( titanAppsUrlPrefix, email ),
+			href: getTitanEmailUrl( titanAppsUrlPrefix, email, false, window.location.href ),
 			image: titanMailIcon,
 			imageAltText: translate( 'Titan Mail icon' ),
+			isInternalLink: true,
 			title: translate( 'View Mail', {
 				comment: 'View the Email application (i.e. the webmail) for Titan',
 			} ),
 			onClick: getTitanClickHandler( 'webmail' ),
 		},
-		{
-			href: getTitanCalendarUrl( titanAppsUrlPrefix, email ),
```

Comment:
We should remove the `getTitanCalendarUrl()` function since it's no longer used. We can still add it back from this pull request once Titan is able to support it.